dags/multipod/maxtext_v4_configs_aot_hybridsim.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A DAG to run AOT compilation and HybridSim tests for MaxText model configs on TPU v4.
+"""
+import datetime
+from airflow import models
+from dags import composer_env, test_owner
+from dags.vm_resource import TpuVersion, Zone, DockerImage, ClusterName, Project
+from dags.multipod.configs import gke_config
+from xlml.utils import name_format
+from airflow.utils.task_group import TaskGroup
+from dags.multipod.configs import gke_config
+
+# Run once a day at 10 am UTC (2 am PST / 3 am PDT)
+SCHEDULED_TIME = "0 10 * * *" if composer_env.is_prod_env() else None
+
+with models.DAG(
+    dag_id="maxtext_v4_configs_aot_hybridsim",
+    schedule=SCHEDULED_TIME,
+    tags=["multipod_team", "maxtext", "stable", "nightly"],
```

Comment:
We cna drop the `nightly` tag, since this only uses the stable image

Reply:
Done.